### ISSUE TITLE
mj-column: Incorrect width calculation in mobile view when fixed width in pixel is specified

### ISSUE THREAD

**Issue by surajpoddar16** [mjmlio/mjml#2489]
**Describe the bug**
If a fixed width is specified in mj-column, the column gets rendered correctly in desktop view, but in mobile view the formula used for width percentage seems wrong

https://github.com/mjmlio/mjml/blob/master/packages/mjml-column/src/index.js#L141

**Expected behavior**

return `${parsedWidth * 100 / parseInt(containerWidth, 10)}%`

**MJML environment (please complete the following information):**
 - OS: MacOS
 - MJML Version 4.5.1, 4.12.0
 - MJML tool used npm package

**Email sending environment(for rendering issues)**:
 - Platform used to send the email [e.g [Putsmail](https://putsmail.com/)]  

**Affected email clients (for rendering issues):**
 - Email Client [e.g Gmail] 
 - OS: [e.g. Windows]
 - Browser [e.g. Google Chrome]

**Screenshots**
<img width="1254" alt="Screenshot 2022-06-28 at 8 01 13 PM" src="https://user-images.githubusercontent.com/4719295/176205291-bd763fa3-a333-4f9a-893a-e311046b6e0b.png">

**Comment by surajpoddar16** [comment]
Above the mobile view breakpoint, the fixed pixel value is respected, the problem occurs below the mobile breakpoint.

**Comment by iRyusa** [comment]
Yup because it's only on the mobile width that you have this issue. You can open a PR on this if you want

**Comment by PumpedSardines** [comment]
Why was #2507 closed?

**Comment by sneko** [comment]
I can confirm this issue.

I switched to percentage for columns as mentioned above. But still, I do think this could be solved because having a weird tiny percentage for no reason makes no sense. And if columns requires a %, the compilation should fail when using pixels.

**Comment by dazgreer** [comment]
Closing as this appears to have been resolved since the original report.
